Docs Enhancements To CLI Section (#689)
spiceai/docs: spiceaidocs/docs/cli/reference/connect.md

pagination_next: null
---

> Connect to an app on Spice's Cloud Platform.

Reviewer: ```suggestion
Connect to an app on the Spice.ai Cloud Platform.
```

### Requirements

> - Authentication to Spice's Cloud Platform via [`login`](/cli/reference/login)

Reviewer: ```suggestion
- Authenticated to Spice.ai Cloud Platform via [`login`](/cli/reference/login)
```

### Usage

```shell

> spice connect [path] [flags]

Reviewer: ```suggestion
spice connect [app] [flags]
```

```

> - `path`: Path to project.

Reviewer: ```suggestion
- `app`: The app slug in the Spice.ai Cloud Platform (i.e. `spiceai/tpch`).
```
